lib/kubernetes-deploy/kubernetes_resource/horizontal_pod_autoscaler.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class HorizontalPodAutoscaler < KubernetesResource
+    TIMEOUT = 5.minutes
```

Comment:
Is this arbitrary or based on testing? 30s seemed short to me, but this seems really long 😄